cfme/configure/access_control.py

```diff
@@ -1,5 +1,6 @@
 from navmazing import NavigateToSibling, NavigateToAttribute
-from widgetastic_manageiq import UpDownSelect, PaginationPane, SummaryFormItem, Table
+from widgetastic_manageiq import UpDownSelect, PaginationPane, SummaryFormItem, Table, \
+    BaseListEntity
```

Comment:
**REQUIRED**
Please follow the style guide

Reply:
Fixed